lib/renderer/command-line.js

```diff
@@ -0,0 +1,14 @@
+'use strict'
+
+exports.hasSwitch = function (name) {
+  return process.argv.includes(`--${name}`)
```

Comment:
This should be implemented as a call through to the C++ `CommandLine::HasSwitch`, which has more robust logic than this. Same for `getSwitchValue` below.

Reply:
done